### PR TITLE
feat: add tests with single patient and cross patient AQL queries

### PR DESCRIPTION
**Pull request by vladislavploaia**
Add performance tests to cover AQL queries mentioned in the below page:
https://wiki.vitagroup.ag/display/PEN/AQL+Queries
Story: https://jira.vitagroup.ag/browse/CDR-380
Sub-task: https://jira.vitagroup.ag/browse/CDR-422

### CODE REVIEW

File: test-plans/ehrbase_catalonia_rfp_queries.jmx [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+          <hashTree/>
+        </hashTree>
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="Get all patients which have a specific diagnosis for a specific health care facility" enabled="true">
+          <boolProp name="HTTPSampler.postBodyRaw">true</boolProp>
+          <elementProp name="HTTPsampler.Arguments" elementType="Arguments">
+            <collectionProp name="Arguments.arguments">
+              <elementProp name="" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                <stringProp name="Argument.value">{&#xd;
+  &quot;q&quot;: &quot;SELECT DISTINCT e/ehr_id/value FROM EHR e CONTAINS COMPOSITION c CONTAINS EVALUATION ev[openEHR-EHR-EVALUATION.problem_diagnosis.v1] WHERE c/context/health_care_facility/name/value=&apos;Hospital&apos; AND ev/data[at0001]/items[at0002]/value/value=&apos;Problem/Diagnosis name 10&apos;&quot;&#xd;
```
Comment (vmueller-vg) [inline test-plans/ehrbase_catalonia_rfp_queries.jmx:213]: c/context/health_care_facility/name/value='hcf0hcp0'

instead of 'Hospital'

**Review by vmueller-vg — CHANGES_REQUESTED** [review]
General suggestion: Can we add this to the already existing testplan instead of having a separate one?

File: test-plans/ehrbase_catalonia_rfp_queries.jmx (hunk 1 above)
Comment (vladislavploaia) [reply test-plans/ehrbase_catalonia_rfp_queries.jmx:213]: Hello @vmueller-vg,
I updated the query, as it didn't return any results. With 'Hospital' it is returning something.
PFB screenshot:
image
In below query, I selected all _c/context/health_care_facility/name/value_ and seems that no other values are present, except this 'Hospital'.
Query: 
`SELECT DISTINCT e/ehr_id/value, c/context/health_care_facility/name/value FROM EHR e CONTAINS COMPOSITION c CONTAINS EVALUATION ev[openEHR-EHR-EVALUATION.problem_diagnosis.v1] WHERE ev/data[at0001]/items[at0002]/value/value='Problem/Diagnosis name 10'`
image

**Comment by vladislavploaia** [comment]
> General suggestion: Can we add the new queries to the already existing testplan instead of having a separate one?

Fine. These will be integrated in existing test plan.

**Review by vmueller-vg — APPROVED** [review]

**Pull request merged**